tests/string_quotes.py

```diff
@@ -38,6 +38,8 @@
 "x = '''; y = \"\"\"\""
 "x = ''''; y = \"\"\"\"\""
 "x = '' ''; y = \"\"\"\"\""
+'unnecessary \"\"escaping'
```

Comment:
Could you add some in this vein for the opposite case?
`"unnecessary \'\'escaping"`